open-codegen/opengen/main.py

```diff
@@ -9,9 +9,14 @@
 u = cs.SX.sym("u", 5)  # decision variable (nu = 5)
 p = cs.SX.sym("p", 2)  # parameter (np = 2)
 phi = og.functions.rosenbrock(u, p)  # cost function
+c_f1 = cs.vertcat(1.5 * u[0] - u[1], cs.fmax(0.0, u[2] - u[3] + 0.1))
```

Comment:
Since `main.py` is our "playground", you don't need to include this to your commits